### DISCUSSION TITLE
FS22 - Who is still playing on it?

### DISCUSSION THREAD

**Discussion by Darrenishere** [Courseplay/Courseplay_FS22#3611] [General]
As much as wanted to love FS25, not yet. Riverbend Springs seems all washed out and it is just to soon. I'm sticking with FS22 until the modders start getting CoursePlay, Autodrive, etc. Can't wait for Oxy's new Map. Until then.........

**Comment by MrDonnyNoPants** [comment]
I am doing the same thing along with the fact that most of the maps that I will be playing on have not be ported over yet.

**Comment by Misamangame** [comment]
There is no proper map out and long time will not be...

**Comment by rohne83** [comment]
I only use FS25 for modding, but no actual savegame there except for the testing stuff.
Even if AD, CP, betterContracts and easyDecControls might find their way into FS25 soon, it's still lacking huge part of important machinery and a decend map. Exactly what I am working on (part of, but I also rely on many mods from others ofc).
If I want to play, I'll still open my FS22 savegame for the next months.
